src/AltinnCore/AltinnCore.Runtime.Db/Repository/FormDataRepository.cs

```diff
@@ -0,0 +1,118 @@
+using Microsoft.Azure.Documents.Linq;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AltinnCore.Runtime.Db.Repository
+{
+    public class FormDataRepository : IFormDataRepository
+    {
+        private readonly Uri _databaseUri;
+        private readonly Uri _collectionUri;
+        private readonly string databaseId;
+        private readonly string collectionId;
+        private static DocumentClient _client;
+        private readonly AzureCosmosSettings _cosmosettings;
+
+        public FormDataRepository(IOptions<AzureCosmosSettings> cosmosettings)
+        {
+            //Retrieve configuration values from appsettings.json
+            _cosmosettings = cosmosettings.Value;
+            _client = new DocumentClient(new Uri(_cosmosettings.EndpointUri), _cosmosettings.PrimaryKey);
+            _databaseUri = UriFactory.CreateDatabaseUri(_cosmosettings.Database);
+            _collectionUri = UriFactory.CreateDocumentCollectionUri(_cosmosettings.Database, _cosmosettings.Collection);
+            databaseId = _cosmosettings.Database;
+            collectionId = _cosmosettings.Collection;
+            _client.CreateDatabaseIfNotExistsAsync(new Database { Id = _cosmosettings.Database }).GetAwaiter().GetResult();
+            _client.CreateDocumentCollectionIfNotExistsAsync(_databaseUri,
+                new DocumentCollection { Id = _cosmosettings.Collection }).GetAwaiter().GetResult();
+        }
+
+        /// <summary>
+        /// To insert new form data into formdata collection
+        /// </summary>
+        /// <param name="item"></param>
+        /// <returns></returns>
+        public async Task<FormData> InsertFormDataIntoCollectionAsync(FormData item)
+        {
+            try
+            {
+                var document = await _client.CreateDocumentAsync(_collectionUri, item);
+                var res = document.Resource;
+                var formData = JsonConvert.DeserializeObject<FormData>(res.ToString());
+                return formData;
+            }
+            catch (Exception ex)
+            {
+                throw ex;
+            }
+        }
+
+        /// <summary>
+        /// Get the formdata based on the input parameters
+        /// </summary>
+        /// <param name="reporteeId">the id of the reportee</param>
+        /// <param name="reporteeElementId">the id of the reporteeelement</param>
+        /// <param name="formId">the id of the form</param>
+        /// <returns>the form data for the given parameters</returns>
+        public async Task<FormData> GetFormDataFromCollectionAsync(string reporteeId, string reporteeElementId,string formId)
+        {
+            try
+            {
+                string sqlQuery = "SELECT * FROM FORMDATA WHERE FORMDATA.reporteeElementId = '" + reporteeElementId + "' and FORMDATA.formId = '" + formId + "'";
```

Comment:
can't this be written like this: $"SELECT * FROM FORMDATA WHERE FORMDATA.reporteeElementId = '{reporteeElementId}' and FORMDATA.formId = '{formId}'"